travis_ci/SETUP.md

```diff
@@ -2,7 +2,7 @@
 
 ## Necessary Steps
   - Follow instructions to [Obtain a FOSSA API Key](/OBTAINING_API_KEY.md)
-  - Request a Github Admin (see #github-admins Slack Channel) to add the FOSSA API Key to your Travis Project
+  - Request a Github Admin (see #fossa-support or #github-admins Slack Channel) to add the FOSSA API Key to your Travis Project
```

Comment:
now I'm thinking about this, opening the "obtain a fossa api key" link above takes the reader to a page essentially explaining the same thing so I think it'd be better to either
- remove the `OBTAINING_API_KEY.md` file and the line above
- remove this line and update the `OBTAINING_API_KEY.md` file to mention the slack channel instead of specific people